pkg/client/mongodb/mongodb.go

```diff
@@ -0,0 +1,57 @@
+package mongodb
+
+import (
+	"context"
+	"github.com/joho/godotenv"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+	"os"
+	"time"
+)
+
+var DB *mongo.Client = ConnectDB()
+var collection *mongo.Collection
+
+func ConnectDB() *mongo.Client {
+	client, err := mongo.NewClient(options.Client().ApplyURI(os.Getenv("MONGODB_URI")))
+	//ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	//defer cancel()
+	//client, err := mongo.Connect(context.Background(), options.Client().ApplyURI(os.Getenv("MONGODB_URI")))
+	//defer func() {
+	//	if err = client.Disconnect(ctx); err != nil {
+	//		panic(err)
+	//	}
+	//}()
+
+	collection = GetCollection(client, "accounts")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	err = client.Connect(ctx)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
It is not a good idea to stop the execution of a program from a different place than the main file. It makes tracking trickier, and if you think about it, your package describes what it does. It should be up to the caller to decide if creating a mongo connection is grounds for stopping the entire program. I would prefer returning an error almost always, and having caller decide what to do with it